debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -19,6 +19,7 @@ const djdt = {
     handleDragged: false,
     init() {
         const djDebug = getDebugElement();
+        window.djdt.update_on_ajax = djDebug.getAttribute("update-on-ajax") === "True";
```

Comment:
It's not completely consistent but we generally prefer using `camelCase` in JavaScript code.

Also, other parts of the code use the dataset attribute, so this should be `djDebug.dataset.updateOnAjax` and `data-update-on-ajax` (or whatever the name of the attribute is). A `data-` prefix for attributes is a good idea anyway because such attributes are always valid HTML.

I wonder if there is a better place for this property though. Tim suggested using localStorage to allow switching between the auto update and manual update behavior on the fly, and this probably wouldn't work with the way it's proposed here. Also, I don't think we want to expose the attribute to external modification (just yet). Putting it in the same place as the `handleDragged` property may be a good idea?

Reply:
> Tim suggested using localStorage to allow switching between the auto update and manual update behavior on the fly, and this probably wouldn't work with the way it's proposed here.

For some context, when Elineda and I spoke yesterday, we decided we could start with a settings control. If we find that it doesn't meet developers needs, we can add the JS to make it more stateful.

Reply:
Makes sense!

Reply:
I did change asked, but I don't put on a localstorage, I just with handleDragged.